libs/s25main/world/NodeMapBase.h

```diff
@@ -24,14 +24,17 @@
 /// Base class for a map with the geometry of the world (MapBase)
 /// Provides nodes of a given type with accessors to the node accepting flat indices or points
 template<typename T_Node>
-class NodeMapBase final : public MapBase
+class NodeMapBase : public MapBase
```

Comment:
See above. The final makes also the resize final and devirtualize it

Reply:
Right, I changed it back to `final`. Just added a `Map` member function for computing max/min values using standard functions (looping over the vector instead the map).